testplan/common/utils/watcher.py

```diff
@@ -71,7 +72,7 @@ def _get_common_lines(self, data: CoverageData) -> Dict[str, Set[int]]:
     @contextmanager
     def disabled(
         self, logger: Optional[Logger] = None, reason: Optional[str] = None
-    ):
+    ) -> None:
```

Comment:
It returns Generator
ref: https://adamj.eu/tech/2021/07/04/python-type-hints-how-to-type-a-context-manager/

Reply:
Thanks for catching, updating.